src/hidpp20.c

```diff
@@ -99,11 +99,23 @@ hidpp20_sw_led_control_get_mode_string(const enum hidpp20_led_sw_ctrl_led_mode m
 		str = numeric;
 		break;
 	}
-#undef CASE_RETURN_STRING
 
 	return str;
 }
 
+const char*
+hidpp20_get_quirk_string(enum hidpp20_quirk quirk)
+{
+	switch (quirk)
+	{
+	CASE_RETURN_STRING(HIDPP20_QUIRK_NONE);
+	CASE_RETURN_STRING(HIDPP20_QUIRK_G305);
+	}
+#undef CASE_RETURN_STRING
```

Comment:
two things here: `{` needs to be on the same line as the `switch`. And instead of `undef` here, just move the `CASE_RETURN_STRING` into the top-level, i.e. below the `#include` statements so we don't need to undef it.

Having a `define` in one function but the `undef` in another is bad form.